sdk/core/Azure.Core/tests/Azure.Core.Tests.csproj

```diff
@@ -21,17 +21,18 @@
     <PackageReference Include="Azure.Security.KeyVault.Secrets" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="..\src\Shared\Multipart\*.cs" />
-    <Compile Include="..\src\Shared\ArrayBufferWriter.cs" />
-    <Compile Include="..\src\Shared\AzureResourceProviderNamespaceAttribute.cs" />
-    <Compile Include="..\src\Shared\ConnectionString.cs" />
-    <Compile Include="..\src\Shared\ForwardsClientCallsAttribute.cs" />
-    <Compile Include="..\src\Shared\NoBodyResponseOfT.cs" />
-    <Compile Include="..\src\Shared\OperationHelpers.cs" />
-    <Compile Include="..\src\Shared\PageResponseEnumerator.cs" />
-    <Compile Include="..\src\Shared\RetriableStream.cs" />
-    <Compile Include="..\src\Shared\RequestRequestContent.cs" />
-    <Compile Include="..\src\Shared\ValueStopwatch.cs" />
+    <Compile Include="..\src\Shared\Multipart\*.cs" Link="Shared\Multipart\%(RecursiveDir)\%(Filename)%(Extension)" />
```

Comment:
I think you can use LinkBase="Shared"

Reply:
Ugh.  I've been doing this the hard way everywhere else.  Thanks for the pointer!